WooCommerce/Classes/Tools/Zendesk/ZendeskManager.swift

```diff
@@ -156,7 +158,7 @@ class ZendeskManager: NSObject {
 
     /// Displays an alert allowing the user to change their Support email address.
     ///
-    func showSupportEmailPrompt(from controller: UIViewController, completion: @escaping (Bool) -> Void) {
+    func showSupportEmailPrompt(from controller: UIViewController, completion: @escaping UserInformationCompletion) {
```

Comment:
nitpick: another convention we adopted was prefixing Completion handlers with `on`, so this could be `onUserInformationCompletion`

Reply:
updated!